core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -1034,4 +1034,9 @@ default Stream<HostingGoalForTablet> getTabletHostingGoal(final String tableName
     throw new UnsupportedOperationException();
   }
 
+  default Stream<TabletInformation> getTabletInformation(final String tableName, final Range range)
```

Comment:
```suggestion
  /**
   * @return a stream of tablets information for tablets that fall in the specified range
   * @since 4.0.0
   */
  default Stream<TabletInformation> getTabletInformation(final String tableName, final Range range)
```